docs/riff_core_deployer_tail.md

```diff
@@ -8,11 +8,11 @@ watch deployer logs
 
 ### Synopsis
 
-Stream runtime logs for a deployer until canceled. To cancel, press Ctl-c in the
-shell or kill the process.
+Stream runtime logs for a deployer until canceled. To cancel, press Ctl-c in
```

Comment:
This line was exactly 80 characters. Is the test off by one?

Reply:
If a line of width 80 is valid, then:
```
func isValid(doc string) bool {
	lines := strings.Split(doc, "\n")
	for _, line := range lines {
		if len(line) > 80 {
			return false
		}
	}
	return true
}
```
is correct and not `if len(line) >= 80`.

Reply:
But my manual fix was wrong. The doc of this command was marked as invalid because of the line `As new deployer pods are started, the logs are displayed. To show historical logs` (81 chars)

Reply:
The off-by-one error was in my brain 😅
Repushing an update within the next minutes.